servers/rendering/rendering_method.h

```diff
@@ -33,7 +33,14 @@
 
 #include "servers/rendering/storage/render_scene_buffers.h"
 #include "servers/rendering_server.h"
+
+#ifdef _3D_DISABLED
+// RendererSceneCull::render_camera is empty when 3D is disabled, but
+// it and RenderingMethod::render_camera have a parameter for XRInterface.
+#define XRInterface Object
```

Comment:
Should this be `RefCounted` instead of `Object`, since the parameter is actually `Ref<XRInterface>`? I don't know what'll happen if you try to use `Ref<T>` with a type that isn't `RefCounted`.